installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

```diff
@@ -88,6 +104,7 @@ include::modules/ipi-install-following-the-installation.adoc[leveloffset=+1]
 include::modules/ipi-install-verifying-static-ip-address-configuration.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
+[id="additional-resources_creating_manifest_ignition"]
 == Additional resources
-
+* xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installation-user-infra-generate-k8s-manifest-ignition_installing-bare-metal[{product-title} Creating the Kubernetes manifest and Ignition config files]
```

Comment:
You could update "[{product-title} Creating the Kubernetes manifest and Ignition config files]" to be "Creating the Kubernetes manifest and Ignition config files]". "`{product-title}`" is not in the target heading and it does not seem to apply to the sentence.